ivy/functional/frontends/numpy/creation_routines/building_matrices.py

```diff
@@ -36,3 +36,8 @@ def vander(x, N=None, increasing=False):
     elif ivy.is_bool_dtype or ivy.is_int_dtype(x):
         x = x.astype(ivy.int64)
     return ivy.vander(x, N=N, increasing=increasing)
+
+
+# diagflat
+def diagflat(v, k=0):
+    return ivy.diagflat(v, k=k)
```

Comment:
There might be some miscommunication here: ivy function doesn't take `k` but is named `offset` instead, so the whole thing should be:
```
def diagflat(v, k=0):
    return ivy.diagflat(v, offset=k)
```
where this part `def diagflat(v, k=0):` resembles the numpy convention: https://numpy.org/doc/stable/reference/generated/numpy.diagflat.html

and this part `return ivy.diagflat(v, offset=k)` takes the ivy convention, thanks!

Reply:
hey thanks for this.